deepmedic/dataManagement/io.py

```diff
@@ -27,7 +27,7 @@ def loadVolume(filepath):
         img = np.expand_dims(img, axis=2)
     elif len(img.shape) > 3 :
         # 4D volumes could have been given. Often 3Ds are stored as 4Ds with 4th dim == 1.
-        assert img.shape[3] > 1
+        assert img.shape[3] == 1
```

Comment:
This has been hotfixed on master. Should have been pulled. I d suggest each PR sticks with its own changes and pulls the rest from other branches. (eg tensorboard-branch may take a while to merge in master. This hotfix is easy to do in a more fast-to-merge branch, eg straight on dev or master). This will conflict during merge, but easy to resolve. I am mostly saying it for the future, as a practice.